content/docs/command-reference/run.md

```diff
@@ -244,14 +244,9 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   command's code is non-deterministic
   ([not recommended](#avoiding-unexpected-behavior)).
 
-- `--no-commit` - do not save outputs to cache. A stage created, while nothing
-  is added to the cache. In the stage file, the file hash values will be empty;
-  They will be populated the next time this stage is actually executed, or
-  `dvc commit` can be used to force committing existing output file versions to
-  cache.
-
-  This is useful to avoid caching unnecessary data repeatedly when running
-  multiple experiments.
+- `--no-commit` - do not store outputs in the cache (`dvc.yaml` and `dvc.lock`
+  are still created or updated); useful to avoid caching unnecessary data when
```

Comment:
`useful to avoid caching unnecessary data`

this sounds like `-O` options for an output

Reply:
True. But I guess its the -O/M options that need some clarification... Updated in , PTAL.

Reply:
* updated in 3711dc81

Reply:
okay, looks good